checklists/alz_checklist.en.json

```diff
@@ -2102,7 +2067,7 @@
         {
             "category": "Management",
             "subcategory": "Data Protection",
-            "text": "Consider cross-region replication in Azure for BCDR with paired regions",
```

Comment:
Too vague. Is this for storage? OK, then is correct, otherwise paired region reference is incorrect since other services can CRR between non-paired regions.